lib/hammer_cli/exceptions.rb

```diff
@@ -5,5 +5,6 @@ class OperationNotSupportedError < StandardError; end
   class ModuleLoadingError < StandardError; end
   class ModuleDisabledButRequired < StandardError; end
   class NoCACertificate < StandardError; end
+  class MissingAPIError < StandardError; end
```

Comment:
The error name is a bit misleading. Could you rename it to something like `DataKeyMissingError`?